src/components/SettingsCard/SettingsCard.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import React from 'react'
+
+import { Link } from 'react-router'
+import { JIcon, JText } from 'components/base'
+
+type Props = {
+  title: string,
+  description: string,
+  path: string,
+  iconName?: string,
+  iconColor?: 'white' | 'blue' | 'gray' | 'sky',
+  iconSize?: 'small' | 'medium' | 'large' | 'xlarge',
+}
+
+SettingsCard.defaultProps = {
+  iconName: 'setting',
+  iconColor: 'gray',
+  iconSize: 'medium',
+}
```

Comment:
if icon props could be excluded from component props, why we set some default values for them?
all of them could be null.
And before using of them, we should check `iconName && iconColor && iconSize && <...>`

Reply:
but if we don't have setting cards without icons, we should make these props required

Reply:
I think, here the `size` is always medium. Isn't it?

Reply:
Both of you are right: `iconName` is should be required, and `iconSize` always is should has `medium` value, only color of icon has options.